util/wren.mk

```diff
@@ -103,9 +103,13 @@ else
 	SHARED_LIB_FLAGS := -Wl,-soname,$@.so
 	SHARED_EXT := so
 
+ifeq (,$(findstring Windows,$(OS)))
```

Comment:
This doesn't seem to be doing the right thing on Travis. Would `ifeq ($(OS),mingw32)` work here like some of the other Windows-specific tests use?